src/libsystemd-network/sd-ndisc.c

```diff
@@ -251,7 +251,7 @@ static int ndisc_handle_router(sd_ndisc *nd, ICMP6Packet *packet) {
                 log_ndisc(nd, "Received Router Advertisement from %s: flags=0x%0*"PRIx64"%s%s%s, preference=%s, lifetime=%s",
                           IN6_ADDR_TO_STRING(&a),
                           flags & UINT64_C(0x00ffffffffffff00) ? 14 : 2, flags, /* suppress too many zeros if no extension */
-                          s ? " (" : "", s, s ? ")" : "",
+                          s ? " (" : "", strempty(s), s ? ")" : "",
```

Comment:
Not that it matters much but to me it seems weird that `ndisc_router_flags_to_string` returns `NULL` instead of an empty string there.

Reply:
As, I'd like not to use `isempty()` here.
We some _to_string() functions return NULL Iin such case, and some return an empty string. No strict rule there, I think.

Reply:
I think in terms of looking for fishy stuff in the logs it would be better to show "unknown"/"-" (or whatever). This way it wouldn't be necessary to use `isempty()/s ?` and remember that it can return either a string or `NULL` on success. Then again it doesn't matter much.

Reply:
OK. The suggestion is applied. Thank you.